app/furniture/marketplace/breadcrumbs.rb

```diff
@@ -27,6 +27,23 @@
   link t("marketplace.orders.index.link_to"), marketplace.location(child: :orders)
 end
 
+crumb :marketplace_order_notification_methods do |marketplace|
+  parent :edit_marketplace, marketplace
+  link t("marketplace.order.notification_methods.index.link_to"), marketplace.location(child: :order_notification_methods)
+end
+
+crumb :new_marketplace_order_notification_method do |order_notification_method|
+  parent :marketplace_order_notification_methods, order_notification_method.marketplace
+  link t("marketplace.order.notification_methods.index.link_to"),
```

Comment:
Isn't this supposed to be?
```suggestion
  link t("marketplace.order.notification_methods.new.link_to"),
```